blog/apps.py

```diff
@@ -0,0 +1,10 @@
+"""Configures the Blog app."""
```

Comment:
Same here.

Reply:
I've to add a more detailed description, right?

Reply:
Marking as resolved.